From a8b1079b34dcde9f7a812644ad581a4f5aa447db Mon Sep 17 00:00:00 2001
From: Vinay Simha BN <vinaysimha@inforcecomputing.com>
Date: Mon, 4 Jan 2016 20:13:07 +0530
Subject: [PATCH 05/17] leds: bt and wlan configuration for ifc6309

DS3 - BT LED   - PMIC - MPP_03
DS2 - WLAN LED - PMIC - MPP_02

Signed-off-by: Vinay Simha BN <vinaysimha@inforcecomputing.com>
---
 arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi b/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
index cfcbfee..2ec5235 100644
--- a/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
+++ b/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
@@ -248,15 +248,15 @@
 
 			led@5 {
 				label = "apq8016-sbc:yellow:wlan";
-				gpios = <&pm8916_mpps 2 GPIO_ACTIVE_HIGH>;
-				linux,default-trigger = "wlan";
+				gpios = <&pm8916_mpps 3 GPIO_ACTIVE_HIGH>;
+				linux,default-trigger = "rfkill0";
 				default-state = "off";
 			};
 
 			led@6 {
 				label = "apq8016-sbc:blue:bt";
-				gpios = <&pm8916_mpps 3 GPIO_ACTIVE_HIGH>;
-				linux,default-trigger = "bt";
+				gpios = <&pm8916_mpps 2 GPIO_ACTIVE_HIGH>;
+				linux,default-trigger = "rfkill1";
 				default-state = "off";
 			};
 		};
-- 
1.9.1

